sale_order_digitized_signature/models/sale.py

```diff
@@ -9,7 +9,7 @@ class SaleOrder(models.Model):
 
     customer_signature = fields.Binary(
         string='Customer acceptance',
-    )
+        attachment=True)
```

Comment:
Please include migration script for this. You can copy the scheme done in https://github.com/OCA/bank-payment/tree/12.0/account_banking_mandate/migrations/12.0.2.0.0